.github/workflows/pull_request.yml

```diff
@@ -36,18 +36,13 @@ jobs:
       - uses: actions/setup-node@v3
         with:
           node-version-file : ".nvmrc"
-      - name: Get yarn cache
-        id: get-yarn-cache
-        run: echo "dir=$(yarn cache dir)" >> $GITHUB_OUTPUT
-      - name: Cache yarn modules
-        id: cache-yarn
-        uses: actions/cache@v3
-        with:
-          path: ${{ steps.get-yarn-cache.outputs.dir }}
-          key: ${{ runner.os }}-yarn-${{ hashFiles('yarn.lock') }}
-      - name: Install yarn deps
-        if: steps.cache-yarn.outputs.cache-hit != 'true'
-        run: yarn
+          cache: "npm"
+      - name: Get npm cache
+        id: get-npm-cache
+        run: echo "dir=$(npm get cache)" >> $GITHUB_OUTPUT
```

Comment:
This should no longer be needed.

Reply:
Just to help my understanding, is this no longer required as the caching is handled in `setup-node`?

Reply:
Correct